src/components/widgets/accounts-ratios/accounts-ratios.tmpl.html

```diff
@@ -0,0 +1,100 @@
+            {{ calculatedNumerator | mnoCurrency : widget.content.layout.currency }}
+          </div>
+
+          <div class="legend">
+            {{ widget.content.layout.numerator }}
+          </div>
+
+          <div setting-offsets ng-show='simulationMode' parent-widget="widget" deferred="::numeratorOffsetsDeferred" offsets-key='numerator' initial-offsets='widget.metadata.offset.numerator' currency='widget.content.layout.currency' interval-count='intervalCount' apply-to-intervals="isPnl" />
+        </div>
+
+        <div class="center-block separator" />
+
+        <div class="member">
+          <div class="price" ng-if="!simulationMode">
+            {{ totalDenominator | mnoCurrency : widget.content.layout.currency }}
+          </div>
+
+          <div class="price" ng-if="simulationMode">
+            {{ calculatedDenominator | mnoCurrency : widget.content.layout.currency }}
+          </div>
+
+          <div class="legend">
+            {{ widget.content.layout.denominator }}
+          </div>
+
+          <div setting-offsets ng-show='simulationMode' parent-widget="widget" deferred="::denominatorOffsetsDeferred" offsets-key='denominator' initial-offsets='widget.metadata.offset.denominator' currency='widget.content.layout.currency' interval-count='intervalCount' apply-to-intervals="isPnl" />
+        </div>
+
+        <div class="text-center">
+          <button class="btn btn-sm btn-default" ng-if="simulationMode" ng-click="toggleSimulationMode(true)" title = "Clear changes to simulation">
+            Cancel
+          </button>
+          <button class="btn btn-sm btn-default" ng-if="!simulationMode" ng-click="toggleSimulationMode()" title="Add custom offsets to the ratio calculation">
+            <i class="fa fa-pencil-square-o" />
+            Simulation
+          </button>
+          <button class="btn btn-sm btn-warning" ng-if="simulationMode" ng-click="saveSimulation()" title="Apply simulation">
+            Save
+          </button>
+        </div>
+      </div>
+
+      <!-- Current value / Chart -->
+      <div class="col-md-6 right-panel">
+        <div setting-hist-mode parent-widget="widget" deferred="::histModeDeferred" accounting-behaviour="{{widget.content.layout.accounting_behaviour}}" />
+
+        <div ng-hide="widget.isHistoryMode" class="current">
+          <div class="price">
+             {{ totalRatio | mnoCurrency : '(ratio)' : false }}
+          </div>
+          <div class="legend">{{ widget.content.layout.ratio }}</div>
+        </div>
+
+        <div class="history chart-container" ng-show="widget.isHistoryMode">
+          <div impac-chart draw-trigger="::drawTrigger.promise" deferred="::chartDeferred"></div>
+          <div class="legend">{{ widget.content.layout.ratio }}</div>
+        </div>
+      </div>
+
+    </div>
+
+    <!-- No data found -->
+    <div ng-show="(isDataFound==false)" common-data-not-found on-display-alerts="onDisplayAlerts()" widget-engine="::widget.category" />
```

Comment:
This needs to be updated for the new scoped vars in the common-data-not-found directive.. I thought my search and replace updated the widget generator template :( 

It should be:
```
<div ng-show="(isDataFound==false)" common-data-not-found on-display-alerts="onDisplayAlerts()" endpoint="::widget.category" width="::widget.width" />
```